README.md

```diff
@@ -20,13 +20,23 @@ Since `vwifi` relies on the Linux wireless (IEEE-802.11) subsystem, [iw](https:/
 sudo apt install iw
 ```
 
-If you're going to run the test script(scripts/verify.sh), python3, hostapd and some packages
+If you're going to run the test script(`scripts/verify.sh`), python3, hostapd and some packages
 are necessary:
 ```shell
 sudo apt install python3 python3-pip hostapd
 pip3 install numpy matplotlib
 ```
 
+## Testing environment
+
+<p align="center"><img src="assets/vwifi.png" alt="logo image" width=60%></p>
+
+In our testing environment, we have **one AP and two STA**.
+
+Our goal is to let them communicate to each other, but with a constraint: **STA cannot talk to other STA directly**. They must pass the packets to AP, and then AP will relay the packets to the destination based on the destination MAC address.
```

Comment:
Add more descriptions about this restriction.

Reply:
Sure.